packages/components/src/components/accordion/_accordion.scss

```diff
@@ -15,6 +15,19 @@
 /// @access private
 /// @group accordion
 @mixin accordion {
+  /// Accordion type tokens that can be overwritten
+  /// @type Map
+  /// @access public
+  /// @group accordion
+  $accordion-type-tokens: $tokens !default !global;
+
+  /// Accordion arrow size in rems
+  /// @access private
+  /// @group accordion
+  $accordion-arrow-size: carbon--type-icon-size(
```

Comment:
Do we need to clamp this value to one of: `[16, 20, 24, 32]`?

Reply:
Yes. I talked with @shixiedesign to confirm these font-size and icon size pairings:

```
12 16
14 16
16 20
20 24
24 32
```

Reply:
Maybe we can omit the 12 16 pairing all together? It's not ideal (would want to shrink icon more and we don't have that size). We already have type size mapping of all 4 icon sizes we support.